src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -9269,6 +9282,7 @@ bool Unit::HandleStatModifier(UnitMods unitMod, UnitModifierType modifierType, f
         case UNIT_MOD_DAMAGE_RANGED:       UpdateDamagePhysical(RANGED_ATTACK);  break;
 
         default:
+            ABORT("Not implemented UnitMod %u", unitMod);
```

Comment:
TC_LOG_ERROR("entities.unit", "Not Implemented UnitMod %u", unitMod);
            ABORT();
this will remove warning?